src/oc/web/actions/search.cljs

```diff
@@ -22,11 +22,9 @@
 (defn query
   "Use the search service to query for results."
   [search-query]
-  (if (> (count search-query) 1)
-    (do
-      (active)
-      (api/query (:uuid (dispatcher/org-data)) search-query query-finished))
-    (reset)))
```

Comment:
Without this reset the old search results won't go away if you type in one character. Is that desired?

Reply:
Without the reset you get behavior like this http://recordit.co/oc9tGn9KaH